docs/reference/scripting.rst

```diff
@@ -838,28 +839,34 @@ SelectedArea
 Functions
 ~~~~~~~~~
 
-SelectedArea.set(rect : rect) : void
+SelectedArea.boundingRect() : :ref:`script-rect`
+    Returns a ectangle which surrounds the selected area.
```

Comment:
You've exposed `boundingRect` as a property, so it shouldn't be documented as a function.